src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const correctRobot = {};
+
+  for (const key in robot) {
+    const robotValue = robot[key];
+
+    if (Object.keys(correctRobot).includes(robotValue)) {
```

Comment:
Using `Object.keys(correctRobot).includes(robotValue)` could be inefficient for large objects. Use `correctRobot.hasOwnProperty(robotValue)` instead for better performance.